src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -194,6 +194,8 @@ public async void BeginInvokeDotNetFromJS(string callId, string assemblyName, st
 
         private async Task OnCircuitOpenedAsync(CancellationToken cancellationToken)
         {
+            Log.CircuitOpened(_logger, Circuit.Id);
```

Comment:
What do you think about using a display string here instead of the actual Id. The Id is really long and it's marginally useful for users to know what it's actual value is versus being able to distinguish circuits in some way.

>  Connection up for CfDJ8DP9EAkPGfZAsPNtItRjMVWZzd4_ED00FlFBkVSfA4PoLIQIGWGUpVewG5VAw5opZNYSWQePn-gmwHgsXJyfAir1vmy_9Gl3ty4cjsXNKgw_y_g1sKSvOjFUB22MG7RGChX5opeXICO0VaQcO3KS7GZDDXQXVMu0CLoWoHSpbB8P.

Reply:
This will change with my next PR, so don't worry too much about it. I can shorten it a bit more if necessary.